pytng/pytng.pyx

```diff
@@ -264,6 +331,64 @@ cdef class TNGFile:
         self.step += 1
         return TNGFrame(xyz, time, self.step - 1, box)
 
+    def _init_first_frame_write(self, int64_t n_atoms,
```

Comment:
I don't like such implicit methods. Please rather make it explicit and fail in the `write` method if no *header* information was supplied.